index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main id="main" container>
```

Comment:
- Good job adding an `id` for the main tag, but you forgot to add the `class` keyword before of container 😄 , I noticed that you added styles in the `styles.css` file with this class, please fix it. 🎃